vm-sysroot/README.txt

```diff
@@ -17,6 +17,9 @@ The resulting image is a single partition without any bootloader/kernel/whatsoev
 ./uml-build.sh
 Executes the two previous scripts, using UML to simulate root access. Doesn't require to be launched as root.
 
+./netns-build.sh
```

Comment:
The script is not defined in this patch

Reply:
I know, but I don't want to leave it undocumented.